fastlane/lib/fastlane/helper/gradle_helper.rb

```diff
@@ -36,10 +36,14 @@ def task_available?(task)
 
       private
 
+      def escaped_gradle_path
```

Comment:
Good thoughts. We discussed and settled on the compromise of making `escaped_gradle_path` part of the public API